files/zh-cn/web/html/element/var/index.html

```diff
@@ -6,10 +6,22 @@
   - Web
 translation_of: Web/HTML/Element/var
 ---
+<<<<<<< HEAD
```

Comment:
broken due to rebaase

Reply:
yeah, the merge conflict must have slipped through during the rebase. I will update shortly